integration/src/test/java/org/kie/hacep/PodAsLeaderSnapshotTest.java

```diff
@@ -48,30 +52,62 @@ public void processMessagesAsLeaderAndCreateSnapshotTest() {
                                                     RemoteKieSession.class);
         try {
 
+            final AtomicInteger attempts = new AtomicInteger(0);
+
             //EVENTS TOPIC
-            ConsumerRecords eventsRecords = eventsConsumer.poll(Duration.ofSeconds(5));
-            assertEquals(11, eventsRecords.count()); //1 fireUntilHalt + 10 stock ticket
+            logger.warn("Checks on Events Topic");
```

Comment:
Warn is the level configured in the Kafka appender to write during tests

Reply:
Maybe it is not good settings in tests. I would expect at least info level in tests. Also to increase severity of message only because of strict configuration of logback-test is not good approach I think. I would suggest to decrease log level in configuration file.

Reply:
With a decreased level we could see too much informations due to info messages used at runtime